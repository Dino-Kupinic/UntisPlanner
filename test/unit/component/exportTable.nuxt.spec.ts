import {it, expect, describe} from "vitest"
import {mountSuspended} from "@nuxt/test-utils/runtime"
import {ExportTable} from "#components"

describe("ExportTable", () => {
  it.skip("Display data correctly", async () => {
    const component = await mountSuspended(ExportTable)
    expect(component.html()).toMatchSnapshot()
  })
})
