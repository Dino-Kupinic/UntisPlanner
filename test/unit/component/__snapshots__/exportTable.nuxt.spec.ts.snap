// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ExportTable > Display data correctly 1`] = `
"<div class="flex justify-around mr-3 ml-3">
  <div class="relative overflow-x-auto">
    <table class="min-w-full table-fixed divide-y divide-gray-300 dark:divide-gray-700">
      <thead class="">
        <tr class="">
          <th scope="col" class="ps-4">
            <div class="relative flex items-start">
              <div class="flex items-center h-5"><input type="checkbox" class="h-4 w-4 dark:checked:bg-current dark:checked:border-transparent dark:indeterminate:bg-current dark:indeterminate:border-transparent disabled:opacity-50 disabled:cursor-not-allowed focus:ring-0 focus:ring-transparent focus:ring-offset-transparent form-checkbox rounded bg-white dark:bg-gray-900 border border-gray-300 dark:border-gray-700 focus-visible:ring-2 focus-visible:ring-primary-500 dark:focus-visible:ring-primary-400 focus-visible:ring-offset-2 focus-visible:ring-offset-white dark:focus-visible:ring-offset-gray-900 text-primary-500 dark:text-primary-400" aria-label="Select all" id="nuid-1"></div>
              <!--v-if-->
            </div>
          </th>
          <th scope="col" class="text-left rtl:text-right px-3 py-3.5 text-gray-900 dark:text-white font-semibold text-sm"><span>id</span></th>
          <th scope="col" class="text-left rtl:text-right px-3 py-3.5 text-gray-900 dark:text-white font-semibold text-sm"><span>User</span></th>
          <th scope="col" class="text-left rtl:text-right px-3 py-3.5 text-gray-900 dark:text-white font-semibold text-sm w-1/2"><span>From</span></th>
          <th scope="col" class="text-left rtl:text-right px-3 py-3.5 text-gray-900 dark:text-white font-semibold text-sm"><span>To</span></th>
        </tr>
      </thead>
      <tbody class="divide-y divide-gray-200 dark:divide-gray-800">
        <tr>
          <td colspan="5">
            <div class="flex flex-col items-center justify-center flex-1 px-6 py-14 sm:px-14"><svg data-v-4fa0a2a1="" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" role="img" class="icon w-6 h-6 mx-auto text-gray-400 dark:text-gray-500 mb-4" width="24px" height="24px" viewBox="0 0 20 20">
                <path fill="currentColor" fill-rule="evenodd" d="M10 1c3.866 0 7 1.79 7 4s-3.134 4-7 4s-7-1.79-7-4s3.134-4 7-4m5.694 8.13c.464-.264.91-.583 1.306-.952V10c0 2.21-3.134 4-7 4s-7-1.79-7-4V8.178a7 7 0 0 0 1.306.953C5.838 10.006 7.854 10.5 10 10.5s4.162-.494 5.694-1.37M3 13.179V15c0 2.21 3.134 4 7 4s7-1.79 7-4v-1.822a7 7 0 0 1-1.306.953C14.162 15.006 12.146 15.5 10 15.5s-4.162-.494-5.694-1.37A7.009 7.009 0 0 1 3 13.179" clip-rule="evenodd"></path>
              </svg>
              <p class="text-sm text-center text-gray-900 dark:text-white">No items.</p>
            </div>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>"
`;
